testsuite/grpc-tests/pom.xml

```diff
@@ -82,7 +84,8 @@
                 <resteasy.test.feature.pack.version>${version.org.jboss.resteasy}</resteasy.test.feature.pack.version>
                 <server.test.feature.pack.groupId>org.wildfly</server.test.feature.pack.groupId>
                 <server.test.feature.pack.artifactId>wildfly-preview-feature-pack</server.test.feature.pack.artifactId>
-                <server.test.feature.pack.version>${version.org.wildfly}</server.test.feature.pack.version>
+                <!--server.test.feature.pack.version>${version.org.wildfly}</server.test.feature.pack.version-->
+                <server.test.feature.pack.version>30.0.0.Final</server.test.feature.pack.version>
```

Comment:
We don't need to block this PR on this, but we should probably use 35 here. Once this is merged, I can create a PR to use channels so that we can always use the latest WildFly release.

Reply:
Still todo